java/source/jdbc.rst

```diff
@@ -307,3 +307,167 @@ values to the given scale.
    102    true    100000000030.0000000    some char text      [1,2]
    INT_FIELD1    BOOL_FIELD2    BIGINT_FIELD5    CHAR_FIELD16    LIST_FIELD19
    103    true    10000000003.0000000    some char text      [1]
+
+Write ResultSet to Parquet File
+===============================
+
+In this example, we have the JDBC adapter result and trying to write them
+into a parquet file.
```

Comment:
Hmm, was this specific example requested? I think it would be better to include a minimal read/write parquet example in `dataset.rst` and remove this one. The `jdbc.rst` already has an example for converting `ResultSet` to `VectorSchemaRoot`. What do you think?